public-html/asset/js/main.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function() {
+    $('.nav_btn').click(function() {
+        $('.mobile_nav_items').toggleClass('active');
+    });
+});
```

Comment:
Are you sure this is working, Don't you need to pass jQuery here?

Reply:
Yes I am including JQuery reference in the html/php file before this script gets called